grafana/docker-compose.yml

```diff
@@ -21,6 +21,7 @@ services:
       - GF_AUTH_OKTA_API_URL=${GODADDY_OKTA_USERINFO_URL}
       - GF_AUTH_OKTA_ALLOWED_GROUPS=${ALLOWED_GODADDY_GROUPS}
       - GF_AUTH_OKTA_ROLE_ATTRIBUTE_PATH=${ROLE_ATTRIBUTE_PATH}
+      - GF_SERVER_ROOT_URL=${SERVER_ROOT_URL}
```

Comment:
why is this being set, I do not see any reference in the OAuth and OIDC setup for grafana for this variable

https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/generic-oauth/
https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/okta/